src/test/java/org/mvel2/util/StringAppenderTest.java

```diff
@@ -0,0 +1,109 @@
+package org.mvel2.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mvel2.util.StringAppender;
+
+public class StringAppenderTest {
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testCharConstructor() {
+    final StringAppender stringAppender = new StringAppender('a');
```

Comment:
Why single quotes for constructor argument, 'a', here but double quotes in other tests?

Reply:
Because i'm using a different constructor for this test (that takes chars).